src/js/components/Tabs/__tests__/Tabs-test.js

```diff
@@ -115,4 +115,93 @@ describe('Tabs', () => {
     fireEvent.mouseOut(getByText('Tab 2'));
     expect(container.firstChild).toMatchSnapshot();
   });
+
+  test('should style as disabled', () => {
+    const { container } = render(
+      <Grommet>
+        <Tabs>
+          <Tab title="Enabled Tab">This tab is enabled</Tab>
+          <Tab title="Disabled Tab" disabled>
+            This tab is disabled
+          </Tab>
+        </Tabs>
+      </Grommet>,
+    );
+
+    expect(container.firstChild).toMatchSnapshot();
+  });
+
+  test('should apply custom theme disabled style', () => {
+    const disabledTextColor = 'blue';
+    const disabledBorderBottomColor = 'green';
+    const customTheme = {
+      tab: {
+        border: {
+          disabled: {
+            color: disabledBorderBottomColor,
+          },
+        },
+        disabled: {
+          color: disabledTextColor,
+        },
+      },
+    };
+
+    const { container, getByText } = render(
+      <Grommet theme={customTheme}>
+        <Tabs>
+          <Tab title="Enabled Tab">This tab is enabled</Tab>
+          <Tab title="Disabled Tab" disabled>
+            This tab is disabled
+          </Tab>
+        </Tabs>
+      </Grommet>,
+    );
+
+    expect(container.firstChild).toMatchSnapshot();
+
+    const disabledTab = getByText('Disabled Tab').parentElement;
+    const disabledTabStyle = window.getComputedStyle(disabledTab);
+    expect(disabledTabStyle.color).toBe(disabledTextColor);
+    expect(disabledTabStyle.borderBottomColor).toBe(disabledBorderBottomColor);
+  });
+
+  test(`should apply custom theme disabled style when theme.button.default is 
```

Comment:
Looks like the snapshot needs to be updated for this test, right now it is failing because the snapshot doesn't match.

Reply:
Snapshot updated.